### PR TITLE
Creating a data model

### PR DESCRIPTION
**Pull request by Nasirkhan294**
### In this milestone I:

- [ ] Setup GitHub and related linters (Stylelint and Robocop)
- [ ] Created databases 
- [ ] Migrated databases 
- [ ] Created Four tables (Users, Posts, Comments, Likes)

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    t.datetime 'updated_at', null: false
+    t.index ['author_id'], name: 'index_posts_on_author_id'
+  end
+
+  create_table 'users', force: :cascade do |t|
+    t.string 'name'
+    t.string 'photo'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+  end
```
Comment (mukhammadrizooff) [inline db/schema.rb]: - [ ] [REQUIRED] Good jobs o far! There should be migration files for creating `users`, `posts`, `comments`, `likes` but you create for only `posts` and `users`. Please create migration files for `comments` and `likes` contains about [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) 🙏🏼

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Nasirkhan294, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/schema.rb (hunk 1 above)
Comment (Nasirkhan294) [reply db/schema.rb]: Hi @mukhammadrizooff, 

Thanks for highlighting the requirements. I will update them and submit another evaluation.

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Nasirkhan294, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- Created migration file for `users`, `posts`, `comments` & `likes` ✔️
- Migrate all file to the schema ✔️ 
- Good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**